### PR TITLE
Updated name, added key final step added variation

### PR DESCRIPTION
**Pull request by skwok-vmware**
Clarified that this creates a Proto persona; clarified that it is critical to follow this up with validating the information to create a validated persona; added variation for Machina proto-persona

### CODE REVIEW

File: content/practices/personas/index.md [hunk 1]
```diff
@@ -76,11 +77,23 @@ why:
    **Tip**: It's helpful to have scissors handy in case you need to cut quadrants apart in order to reassemble
    {{% /callout %}}
 
+10. Plan out how to get access to people that this persona represents and the user research activities that you will do to validate this persona. Check out activities labeled Discovery in the Tanzu Practices center for different options. 
+
+
 ### Success/Expected Outcomes
-Success is when you've created a "master persona" and everyone in the room feels like it represents the target audience as best as possible.
+Success is when you've created a "master persona" and everyone in the room feels like it represents the target audience as best as possible. Remember that a critical next step is to then validate this persona with user research such as interviews
```
Comment (joemoore) [inline content/practices/personas/index.md:84]:
```suggestion
Success is when you've created a "master persona" and everyone in the room feels like it represents the target audience as best as possible. Remember that a critical next step is to then validate this persona with [user research](/practices/user-research-introduction-session/) such as interviews
```

File: content/practices/personas/index.md [hunk 2]
```diff
@@ -76,11 +77,23 @@ why:
    **Tip**: It's helpful to have scissors handy in case you need to cut quadrants apart in order to reassemble
    {{% /callout %}}
 
+10. Plan out how to get access to people that this persona represents and the user research activities that you will do to validate this persona. Check out activities labeled Discovery in the Tanzu Practices center for different options.
```
Comment (joemoore) [inline content/practices/personas/index.md:80]:
```suggestion
10. Plan out how to get access to people that this persona represents and the [user research activities](/practices/user-research-introduction-session/) that you will do to validate this persona. Check out activities labeled Discovery in the Tanzu Practices center for different options. 
```

File: content/practices/personas/index.md [hunk 3]
```diff
@@ -76,11 +77,23 @@ why:
    **Tip**: It's helpful to have scissors handy in case you need to cut quadrants apart in order to reassemble
    {{% /callout %}}
 
+10. Plan out how to get access to people that this persona represents and the user research activities that you will do to validate this persona. Check out activities labeled Discovery in the Tanzu Practices center for different options. 
+
+
 ### Success/Expected Outcomes
-Success is when you've created a "master persona" and everyone in the room feels like it represents the target audience as best as possible.
+Success is when you've created a "master persona" and everyone in the room feels like it represents the target audience as best as possible. Remember that a critical next step is to then validate this persona with [user research](/practices/user-research-introduction-session/) such as interviews
```
Comment (joemoore) [inline content/practices/personas/index.md:84]: Let's rephrase "master persona" as something more inclusive, such as "main persona", "primary persona", or similar.

Comment (joemoore) [reply content/practices/personas/index.md:84]: let's update the following as well: 
> 9. Tape together the highest scoring quadrants into one "master persona" and reads it to the group

File: content/practices/personas/index.md [hunk 4]
```diff
@@ -76,11 +77,23 @@ why:
    **Tip**: It's helpful to have scissors handy in case you need to cut quadrants apart in order to reassemble
    {{% /callout %}}
 
+10. Plan out how to get access to people that this persona represents and the [user research activities](/practices/user-research-introduction-session/) that you will do to validate this persona. Check out activities labeled Discovery in the Tanzu Practices center for different options. 
+
+
 ### Success/Expected Outcomes
-Success is when you've created a "master persona" and everyone in the room feels like it represents the target audience as best as possible.
+Success is when you've created a "master persona" and everyone in the room feels like it represents the target audience as best as possible. Remember that a critical next step is to then validate this persona with [user research](/practices/user-research-introduction-session/) such as interviews
```
Comment (skwok-vmware) [inline content/practices/personas/index.md:84]:
```suggestion
Success is when you've created a consolidated proto-persona and everyone in the room feels like it represents the target audience as best as possible. Remember that a critical next step is to then validate this persona with [user research](/practices/user-research-introduction-session/) such as interviews
```

**Comment by joemoore** [comment]
@BrianMMcClain @anthonyvetter Good to merge!

**Review by anthonyvetter — APPROVED** [review]
LGTM

**Pull request merged**